bemani/api/objects/catalog.py

```diff
@@ -165,11 +166,8 @@ def __format_extras(self) -> Dict[str, List[Dict[str, Any]]]:
 
     @property
     def music_version(self) -> int:
-        if self.game == GameConstants.IIDX:
-            if self.omnimix:
-                return self.version + DBConstants.OMNIMIX_VERSION_BUMP
-            else:
-                return self.version
+        if self.omnimix:
```

Comment:
The implication of this change is that it standardizes omnimixes across all games, instead of just for IIDX. I think I'm fine with that. I kept this IIDX-specific because there's no omnimix support for other mixes, so if a remote server requests songs for an omnimix version of another game, they would get nothing. Are we okay with that implication (mapping requests to omnimixes for non-IIDX to empty responses, instead of the base game's responses)?

Reply:
This was one of the changes i made when adding omnimix support for jubeat. I didn't fully understand why it hard assumed iidx

Reply:
Basically it was to provide limited score support of the base game's scores when a game doesn't have omnimix (which was any game outside of IIDX). Its up for debate what we should do here, the spec is intentionally vague about behavior around this and I chose to try to do the best effort response over a technically correct but less useful one.